convert_gvf_to_vcf/utils.py

```diff
@@ -61,25 +34,27 @@ def read_pragma_mapper(pragma_mapper_file):
             pragma_to_vcf_header[pragma] = vcf_header
     return pragma_to_vcf_header
 
-def read_sequence_ontology_symbolic_allele(so_symbolic_allele_file):
-    """ Read in the file containing sequence ontology symbolic allele and returns a dictionary.
-    :param: so_symbolic_allele_file - the file of sequence ontology symbolic alleles.
-    :return: symbolic allele dictionary - symbolic alleles as key and list of variant types as the value.
+def read_mapping_dictionary(mapping_dictionary):
```

Comment:
Name of this function could be improved or could be merged with `convert_gvf_attributes_to_vcf_values` since it is doing for ALT what `convert_gvf_attributes_to_vcf_values` is doing for INFO and FORMAT